src/app/core/app.component.html

```diff
@@ -37,8 +34,9 @@
       </div>
       <alg-top-right-controls [drawLeftBorder]="currentMode === 'normal'"></alg-top-right-controls>
     </div>
-    <div class="right" [ngClass]="{ collapsed: !leftMenuDisplayed, expanded: leftMenuDisplayed, 'with-mode-bar': currentMode !== 'normal' }">
-      <div class="main-content" [ngClass]="{ folded: !headersDisplayed, unfolded: headersDisplayed }">
+    <div class="right"
```

Comment:
In this project:
If it fits on one line:
```<div class="right" [ngClass]=...>```
Otherwise:
```
<div 
  class="right" 
  [ngClass]=...
>
```

This remarks applies everywhere in this file.

Reply:
Do you have a suggestion of HTML formatter which does that?

Reply:
No, sorry. I do it by hand.